src/quantum/config.js

```diff
@@ -23,7 +22,7 @@ const CONFIG = {
     {
       title: 'MotionMark Animometer',
       secondLink:
-        '/quantum/10/windows10-64/raptor-motionmark_animometer-firefox/pgo',
+        '/quantum/10/windows10-64/raptor-motionmark-animometer-firefox/pgo',
```

Comment:
many thing broke in this PR:   I could not tell if it was the tooltip, or if the raw names changed again.  It was the raw names.